src/core/globalstate.ts

```diff
@@ -48,12 +48,9 @@ export class MobXGlobals {
 	 */
 	isRunningReactions = false;
 
-	/**
-	 * List of observables that have changed in a transaction.
-	 * After completing the transaction(s) these atoms will notify their observers.
-	 */
-	changedAtoms: IAtom[] = [];
+	inBatch: number = 0;
```

Comment:
note that the version attribute needs to be bumped, as the globalState object's signature changes, meaning that this mobx version is not compatible with older mobx version in same memory